history/event.go

```diff
@@ -23,6 +23,7 @@ const (
 	EventDeautomate  = "deautomate"
 	EventLock        = "lock"
 	EventUnlock      = "unlock"
+	EventUpdateTag   = "update_tag"
```

Comment:
I foresee there being more kinds of policy, and we could get bogged down in individual events for each. How about we have a PolicyChange event which subsumes the others, and has the previous and new policies. We can attach labels, if necessary, to make it easier to tell whether it represents a transition e.g., locked, automated.